src/components/SplashScreen/index.tsx

```diff
@@ -86,22 +76,6 @@ export const SplashScreen = (): ReactElement => {
     }
   }
 
-  const onContinue = async () => {
-    trackSafeAppEvent(NAVIGATION_EVENTS.OPEN_LOCKING.action, 'opening')
```

Comment:
Do we still need to track this event somewhere?

Reply:
I would say it is fine for now as this button does not exist anymore and opening the Safe app is already tracked in our wallet events